Packs/rasterize/ReleaseNotes/2_0_36.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Rasterize
+
+- Updated the Docker image to: *demisto/chromium:131.0.6778.117810*.
```

Comment:
Verify with @tkatzir before merging.

Reply:
Approved by me.
The only difference is the removal of Python3.9 (Python3.11 is installed by default.)